ONSdigital/rm-reporting
# Fix security issues

Unpins and upgrades packages to fix security issues and keep the repo more up to date.

There were a few warnings generated by these updates which got fixed also.

## How to test
  - Unit tests should pass without warnings
  - After running the acceptance tests to set up the data, you should be able to hit the response chasing and responses dashboard endpoints and get data from it.
(e.g., `http://localhost:8084/reporting-api/v1/response-chasing/download-report/05eb0f73-e71d-4725-9d1c-859be8acf9c6/cb8accda-6118-4d3b-85a3-149e28960c54`)
